mountpoint-s3/src/inode.rs

```diff
@@ -360,6 +360,10 @@ impl Superblock {
             return Err(InodeError::IsDirectory(inode.ino()));
         }
 
+        if !inode.verify_checksum() {
```

Comment:
Maybe we should just make `self.inner.get` do this check every time?

Reply:
Maybe. And/or in `self.inner.lookup` (or it would miss unlink for example)?

Reply:
Done in latest commit.

Reply:
Or maybe instead we should verify the checksum when we get the full key?